inspect_image.go

```diff
@@ -4,8 +4,10 @@ import (
 	"context"
 	"strings"
 
+	"github.com/buildpacks/lifecycle/platform"
+
```

Comment:
Lint should be catching this but we should only have 3 import groups.

- stdlib
- dependencies
- local

IIRC, there's a setting in GoLand/IntelliJ to configure this.

🤔 , does running `make prepare-for-pr` automagically fix this?

Reply:
it seems to have no effect here.